packages/utils/src/get-snark-artifacts/config.ts

```diff
@@ -0,0 +1,87 @@
+import { Artifact, Proof, SnarkArtifacts, Version } from "../types"
+
+const ARTIFACTS_BASE_URL = "https://unpkg.com/@zk-kit"
+
+const getPackageVersions = async (proof: Proof) =>
+    fetch(`${ARTIFACTS_BASE_URL}/${proof}-artifacts`)
+        .then((res) => res.json())
+        .then((data) => Object.keys(data.versions))
+
+export async function GetSnarkArtifactUrls({
+    proof,
+    version
+}: {
+    proof: Proof.EDDSA
+    version?: Version
+}): Promise<SnarkArtifacts>
+export async function GetSnarkArtifactUrls({
+    proof,
+    numberOfInputs,
+    version
+}: {
+    proof: Proof.POSEIDON
+    numberOfInputs: number
+    version?: Version
+}): Promise<SnarkArtifacts>
+export async function GetSnarkArtifactUrls({
```

Comment:
Why are these functions capitalized?

Reply:
To indicate factory functions (comparable to a class constructor).
I can lowercase it

Reply:
Ahhh, as you prefer!